keps/sig-cluster-lifecycle/kubeadm/20190122-Certificates-copy-for-kubeadm-join--control-plane.md

```diff
@@ -325,7 +334,11 @@ affect component version or version skew policy)
 ## Implementation History
 
 - 22 Jan 2019 - first release of this KEP
-- v1.14 implementation (planned)
+- v1.14 implementation as alpha feature without
+  - Extension of the kubeadm config file for allowing usage of pre-generated certificate keys
+  - TokenCleaner enforcement
+  - E2E tests
+- v1.15 (planned) implementation of the changes not included in v1.14  
```

Comment:
Again, let's have the 1.15 changes in the implementation history after the release.